internal/app/init/internal/platform/baremetal/baremetal.go

```diff
@@ -84,5 +84,13 @@ func (b *BareMetal) Prepare(data *userdata.UserData) (err error) {
 // download the necessary bits and write them to a target device
 // nolint: gocyclo, dupl
 func (b *BareMetal) Install(data *userdata.UserData) error {
-	return install.Install(data)
+	var cmdlineBytes []byte
+	cmdlineBytes, err = kernel.ReadProcCmdline()
+	if err != nil {
+		return err
+	}
+	if err = install.Install(string(cmdlineBytes), data); err != nil {
+		return errors.Wrap(err, "failed to install to bare metal")
+	}
+
```

Comment:
The thought with these changes is that in anticipation of #453 we will need to support providing different kernel params in the `APPEND` field of `extlinux.conf`.